### PR TITLE
Add MozFest homepage and primary page to Percy test

### LINKED ISSUE #3268: Run visual regression (Percy/Cypress) testing on Mozfest templates
**Describe the bug**
When Percy runs, it should check the main Mozfest site templates for regressions. This probably requires extra tests written [here](https://github.com/mozilla/foundation.mozilla.org/blob/master/cypress/integration/endpoint-tests.js)

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #3268

after tons of trials and errors and digging into various docs... THIS FINALLY WORKS!!! 😭

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,10 +10,12 @@ python:
 
 addons:
   postgresql: "9.6"
+  hosts:
```
Comment (mmmavis) [inline .travis.yml]: https://docs.travis-ci.com/user/hosts/

File: cypress.json [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {
   "baseUrl": "http://localhost:8000",
-  "video": false
+  "video": false,
+  "env": {
```
Comment (mmmavis) [inline cypress.json]: https://docs.cypress.io/guides/guides/environment-variables.html#Option-1-configuration-file

**Comment by Pomax** [comment]
tagging cade as second pair of eyes on the travis.yml changes

**Comment by mmmavis** [comment]
🎉

**Pull request merged**